packages/common/package.json

```diff
@@ -32,6 +34,26 @@
     "react": "^17.0.1",
     "react-i18next": "^11.7.3",
     "react-query": "^3.39.2",
-    "react-router": "5.2.0"
+    "react-router": "5.2.0",
+    "react-router-dom": "5.2.0"
+  },
+  "devDependencies": {
+    "@storybook/addon-essentials": "^7.0.14",
+    "@storybook/addon-interactions": "^7.0.14",
+    "@storybook/addon-links": "^7.0.14",
+    "@storybook/blocks": "^7.0.14",
+    "@storybook/preset-scss": "^1.0.3",
+    "@storybook/react": "^7.0.14",
+    "@storybook/react-webpack5": "^7.0.14",
+    "@storybook/testing-library": "^0.0.14-next.2",
+    "eslint-plugin-storybook": "^0.6.12",
+    "prop-types": "^15.8.1",
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
+    "sass": "^1.63.3",
+    "storybook": "^7.0.14"
+  },
+  "dependencies": {
```

Comment:
@sgratch we should not depend on story book in our release code

can you move this so we will not deploy the storybook code to users ?

Reply:
Done.